src/style.css

```diff
@@ -0,0 +1,124 @@
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap);
+
+* {
+  margin: 0;
+  font-family: Roboto , sans-serif;
+  font-style: normal;
```

Comment:
Only the `border-box` property is usually set on the this selector
It's better to set font-family in html selectorMove it to another class
Remove redundant default value `font-style: normal`